doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
 
-Workaround: None.
 
-endif::[]
+.When you change a decision table header in the properties panel, the change is not saved [https://issues.redhat.com/browse/RHDM-1181[RHDM-1181]]
 
-== High availability
+Issue: In the DMN designer, when editing a decision table, you can click a header to view its properties panel. However, if you change settings for the header in the properties panel, the change is not saved.
 
-//approved
-.In a high-availability authoring environment, when one user imports a project, another user is unable to see it [https://issues.jboss.org/browse/RHPAM-2470[RHPAM-2470]]
+Steps to reproduce:
 
-Issue: When multiple users connect to a high-availability {CENTRAL} and one user creates a project, another user cannot see the project.
+. Edit a decision table in an element in the DMN designer.
+. Click a header in the table.
+. Change a field, such as *Input expression*, in the properties panel on the right side of the screen.
+
+Expected result: The change is reflected in the table.
+
+Actual result: The change is not reflected in the table.
+
+Workaround: Click the header to view the pop-up window next to the header. Edit the settings in this window.
+
+.When you open a DMN diagram without layout information, nodes are all in the same position[https://issues.redhat.com/browse/RHDM-1150[RHDM-1150]]
+
+Issue: When you open a DMN model file and this file does not contain layout information, all DMN nodes are displayed in the same position.
 
 Steps to reproduce:
 
-//approved
-. Log in to {CENTRAL} as two different users (A and B) from two different hosts or browsers.
-. As user A and as user B, open the same space.
-. As user A, import a project using an external Git repository URL.
+. Open or create a project in {CENTRAL}.
+. Import a DMN model file that does not contain layout information.
 
-Expected result: As user B, you can see the imported project in the space.
+Expected result: The nodes are spread across the canvas.
 
-Actual result: As user B, you cannot see the imported project in the space.
+Actual result: All nodes are in the same position.
 
-Workaround: As user B, reload the space.
+Workaround: Click the image:ReleaseNotes/automatic_layout.png[] *Automatic layout* button in the toolbar.
 
-//approved
-.In a high-availability authoring environment, role changes in {CENTRAL} are not saved [https://issues.jboss.org/browse/RHPAM-2342[RHPAM-2342]]
 
-Issue: If you configure a role change in high-availability {CENTRAL}, the change can be lost after a time and the roles reset to defaults.
+ifdef::PAM[]
 
-Workaround: Configure the role change again.
+== Process designer
 
-//approved
-.In a high-availability authoring environment, an imported project in {CENTRAL} appears multiple times [https://issues.jboss.org/browse/RHPAM-2372[RHPAM-2372]]
+.If the properties panel is opened, after you maximize and restore the panel, it closes [https://issues.redhat.com/browse/RHPAM-2613[RHPAM-2613]]
```

Comment:
If the process designer *Properties* panel is open and you maximize and then restore the process designer window, the *Properties* panel is closed in the restored window.

Reply:
"the process designer window" is wrong. It is the panel that is maximized and restored within the window, using BC methods.

Reply:
Maybe I misunderstood you here - might be worthy of a BJN even